research/picai/nnunet_scripts/README.md

```diff
@@ -8,3 +26,19 @@ sbatch nnunet_launch.slrm DATASET_NAME UNET_CONFIG FOLD VENV_PATH PLANS_IDENTIFI
 ```
 sbatch nnunet_launch_fold_experiment.slrm DATASET_NAME UNET_CONFIG VENV_PATH PLANS_IDENTIFIER PRETRAINED_WEIGHTS SOURCE_DATASET_NAME SOURCE_PLANS_IDENTIFIER
 ```
+
+## Transfer Learning with nnUNet
+
+The transfer_train.py script provides an easy way to automate transfer learning with nnUNet models. Run the script with the `--help` flag for a list of arguments that can be passed to the script.
```

Comment:
This is a great addition! One last ask is to maybe include an example invocation with only the required args